git_creation_and_tracking/git_creation_and_tracking.md

```diff
@@ -59,6 +61,10 @@ Before you start this module, make sure you
 
 Open a new command line interface on your computer. All exercises in this module will be done in this interface.
 
+<div class = "warning">
+If you are on a Windows machine, make sure you are in the Git Bash application. The Command Prompt (cmd) will not recognize the commands we are using in this module.  
```

Comment:
Add a download link to Git download for windows, which includes git bash?  Perhaps https://git-scm.com/download/win.  Indicate that git bash is part of the windows install and maybe have a quick screen shot of what the icon looks like etc.